case-lib/hijack.sh

```diff
@@ -77,6 +78,7 @@ function exit()
         ;;
     esac
 
+    trap '' EXIT
     builtin exit $exit_status
```

Comment:
We trapped exit, I think the `builtin exit $exit_status` may cause recursive call of exit.

Reply:
here trap is store the `exit`

Reply:
in my test, it is not the recursive call, so this line removed